src/AccessToken.php

```diff
@@ -257,13 +258,12 @@ private function verifyRs256($token, array $certs, $audience = null, $issuer = n
             ]);
 
             // create an array of key IDs to certs for the JWT library
-            $keys[$cert['kid']] =  $rsa->getPublicKey();
+            $keys[$cert['kid']] = new Key($rsa->getPublicKey(), 'RS256');
         }
 
         $payload = $this->callJwtStatic('decode', [
             $token,
             $keys,
```

Comment:
```suggestion
            $keys
```

Just a minor thing, don't feel pressured to accept as I understand this can go either way.

Reply:
definitely a preference thing. I think I lean more towards leaving trailing commas on lists of array items which span multiple lines